src/test/suite/index.ts

```diff
@@ -30,7 +30,6 @@ export function run(): Promise<void> {
                     }
                 });
             } catch (err) {
-                console.error(err);
```

Comment:
If we're removing this, do we need the try-catch block here, or will the error be propagated anyway? Though maybe it's more convenient to have it for future changes.